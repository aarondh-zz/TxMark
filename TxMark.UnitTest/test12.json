{
  "array": [
    "one",
    {
      "one": "one",
      "two": "two",
      "three": {
        "one": "one",
        "two": "two",
        "three": "three",
        "four": {
          "one": "one",
          "two": "two",
          "three": "three",
          "four": "four",
          "five": "five",
          "six": "six"
        },
        "five": "five",
        "six": "six"
      },
      "four": "four",
      "five": "five",
      "six": "six"
    },
    "three",
    "four",
    "five",
    "six"
  ]
}
