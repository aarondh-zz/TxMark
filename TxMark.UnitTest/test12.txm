<div>
  **This should be five:** (print: five of four of three of 2nd of $array)
</div>
