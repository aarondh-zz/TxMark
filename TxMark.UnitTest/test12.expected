<div>
<strong>This should be five:</strong> five
</div>
